### PR TITLE
Email password login UI/UX update

### LINKED ISSUE #205: UI/UX - Align buttons on the protected/feat/email-password-login branch to be the same size as the input field

### PR DESCRIPTION
**Pull request by Sbonelo01**
Related issues: [https://github.com/Umuzi-org/Tilde/issues/205]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

@Vuyisanani , I have implemented the changes that you have requested - everything except the buttons. I think we should keep the current button shapes.
Screenshot from 2021-10-18 01-06-22

### CODE REVIEW

File: frontend/src/components/regions/Login/Presentation.js [hunk 1]
```diff
@@ -1,49 +1,115 @@
...
-        variant="contained"
-        onClick={handleLoginWithGoogle}
-      >
-        Login with Google
-      </Button>
-    </Paper>
+        <Link
+          style={{ overflow: "hidden", whiteSpace: "nowrap" }}
+          underline="always"
+          href="#"
```
Comment (Vuyisanani) [inline frontend/src/components/regions/Login/Presentation.js:91]: Does this link redirects to anywhere?

**Review by babalwa-mbolekwa — CHANGES_REQUESTED** [review]
Hey Sbonelo 
- please fix the font family of `Forgot Password?` to be the same as the other fonts.
- please also reduce the space between the Tick box and the `Remember me` text so that it shows they are grouped
login

Everything else LGTM!

**Comment by Sbonelo01** [comment]
Screenshot from 2021-10-20 21-37-08

**Review by babalwa-mbolekwa — DISMISSED** [review]

File: frontend/src/components/regions/Login/Presentation.js [hunk 2]
```diff
@@ -51,11 +62,48 @@ const LoginForm = ({ loading, error, handleLoginWithGoogle }) => {
...
+              style={{ marginLeft: "0", width: "20px", padding: 0 }}
+              name="checkedB"
+              color="primary"
+            />
+          }
+          label="Remember me"
+        />
+
+        <Link
+          style={{
```
Comment (Vuyisanani) [inline frontend/src/components/regions/Login/Presentation.js]: I think would be better to have the Link styles in a class

File: frontend/src/components/regions/Login/Presentation.js [hunk 3]
```diff
@@ -51,11 +62,48 @@ const LoginForm = ({ loading, error, handleLoginWithGoogle }) => {
       <LockRoundedIcon />
       <TextField label="Email" variant="outlined" type="email" required />
       <TextField label="Password" variant="outlined" type="password" required />
+      <Box m={0}>
+        <FormControlLabel
+          control={
+            <Checkbox
+              style={{ marginLeft: "0", width: "20px", padding: 0 }}
```
Comment (Vuyisanani) [inline frontend/src/components/regions/Login/Presentation.js]: Can we also just add these in a class

**Review by kaleem99 — DISMISSED** [review]

**Review by Vuyisanani — DISMISSED** [review]

File: frontend/src/components/regions/Login/Presentation.js (hunk 1 above)
Comment (Sbonelo01) [reply frontend/src/components/regions/Login/Presentation.js:91]: Hi there @Vuyisanani , I think there is/was an issue specifically for this

File: frontend/src/components/regions/Login/Presentation.js (hunk 2 above)
Comment (Sbonelo01) [reply frontend/src/components/regions/Login/Presentation.js]: sorted

File: frontend/src/components/regions/Login/Presentation.js (hunk 3 above)
Comment (Sbonelo01) [reply frontend/src/components/regions/Login/Presentation.js]: sorted

**Review by Vuyisanani — APPROVED** [review]

File: frontend/src/components/regions/Login/Presentation.js (hunk 1 above)
Comment (Vuyisanani) [reply frontend/src/components/regions/Login/Presentation.js:91]: Oh cool, then a new issue will need to be created for this.

**Review by Vuyisanani — DISMISSED** [review]

**Review by babalwa-mbolekwa — DISMISSED** [review]

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Please overcome the merge conflicts. This can't be merged

**Comment by Vuyisanani** [comment]
Please do not forget to add e.preventDefault()

**Review by Vuyisanani — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**